src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -469,9 +495,10 @@ class GroupReadsByUmi
     in.iterator
       .filter(r => !r.secondary && !r.supplementary)
       .filter(r => (includeNonPfReads || r.pf)                                      || { filteredNonPf += 1; false })
-      .filter(r => (r.mapped && (r.unpaired || r.mateMapped))                       || { filteredPoorAlignment += 1; false })
+      .filter(r => (r.mapped || (r.paired || r.mateMapped))                         || { filteredPoorAlignment += 1; false })
```

Comment:
Somethings wrong here
1. I am a bit confused by the remaining braces here.  Do we need them since we have only OR logical relationships?  
2. if `r.paired` is `false`, then how can `r.mateMapped` every be true? 

I think you want "either the read is mapped, or it's paired and its mate is mapped"
```suggestion
      .filter(r => (r.mapped || (r.paired && r.mateMapped))                         || { filteredPoorAlignment += 1; false })
```

Reply:
Good catch, thanks!